apps/apple/ios/FluentUIReactNativeTester_iOS.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should probably .gitignore this file?

Reply:
It should be already with `**/project.xcworkspace/contents.xcworkspacedata`.  Something might be off with the formatting though